packages/honeycomb-layout/package.json

```diff
@@ -8,8 +8,8 @@
     "clean": "rimraf pids logs coverage dist",
     "lint": "eslint --format=node_modules/eslint-formatter-pretty .",
     "start": "npm run start:production",
-    "start:development": "pm2-dev start .config/pm2.development.json",
-    "start:production": "pm2 start .config/pm2.production.json",
+    "start:development": "cross-env NODE_ENV=development nodemon src/index.js --exec babel-node --watch src/ --ext js",
+    "start:production": "cross-env NODE_ENV=production node dist/index.js",
```

Comment:
We should think about `set NODE_ENV manual via cli`. But I think at the moment it's 👌 when we move the small package `cross-env` to a production dependency (9a350ea).  Or?